.travis.yml

```diff
@@ -10,10 +10,11 @@ cache:
   - tmp/
   - ".sass-cache"
 before_install:
-- nvm install 4.2.6
+- nvm install 6.10.0
 install:
 - bundle install --jobs=3 --retry=3 --deployment --path=${BUNDLE_PATH:-vendor/bundle}
 - npm install
+- npm rebuild node-sass # Remove this soon
```

Comment:
When you install node-sass you build a binary. We cache this binary. When you update node that binary needs rebuilding. :persevere: